compiler/src/utils/config.re

```diff
@@ -449,15 +398,11 @@ let import_memory =
     false,
   );
 
-let compilation_mode =
-  opt(
+let runtime_mode =
+  toggle_flag(
     ~names=["compilation-mode"],
-    ~conv=
-      option_conv(
-        Cmdliner.Arg.enum([("normal", "normal"), ("runtime", "runtime")]),
-      ),
     ~doc="Compilation mode (advanced use only)",
-    None,
+    false,
   );
```

Comment:
I don't understand what this is doing

Reply:
You mean you don't understand why a flag is used or why this variable exists at all?

Reply:
I don't understand why this exist as a cmdliner flag at all considering this should only be used as an attribute now, right?

Reply:
Yeah, I've been trying to get it to work without it being a CLI flag but it seems like the value is being reset at some point during compilation of multiple files that I haven't been able to pin down yet.